### ISSUE TITLE
Please don't open Output window when VSCode start

### ISSUE THREAD

**Issue by unional** [mkloubert/vs-deploy#7]
It pops into the face which is quite annoyed. 🌷

**Comment by mkloubert** [comment]
**Since version 3.5.0** you now have the 2 options `openOutputOnStartup` (default: `(false)`) and `openOutputOnDeploy` (default: `(true)`):

```json
{
    "deploy": {
        "openOutputOnDeploy": true,
        "openOutputOnStartup": false
    }
}
```

You do not have to set `openOutputOnStartup` explicitly. The output window is NOT opened at startup by default now.

**Comment by unional** [comment]
Nice. Upgrading. :)

**Comment by unional** [comment]
Actually it is defaulted to `true`. Setting it to false fixed it.

**Comment by mkloubert** [comment]
Should be **fixed since version 3.5.1**.

It was because of the default settings for `settings.json` file in `package.json`.
